types/react/index.d.ts

```diff
@@ -352,11 +372,38 @@ declare namespace React {
     type ReactFragment = Iterable<ReactNode>;
 
     /**
-     * For internal usage only.
```

Comment:
@mattpocock Do you remember why you removed this? This type is for internal usage only.

Reply:
I think I replaced it with an @internal tag which was rejected in CR, so I removed this to be consistent with the feedback I got.